sdks/common/fn-api/src/test/resources/org/apache/beam/fn/v1/standard_coders.yaml

```diff
@@ -62,6 +62,18 @@ coder:
   urn: "beam:coders:kv:0.1"
   components: [{urn: "beam:coders:bytes:0.1"},
                {urn: "beam:coders:bytes:0.1"}]
+nested: false
```

Comment:
Since the core SDK should not depend on the Fn model or Fn harness, the right place for this is something like `$ROOT/model/src/test/resources/org/apache/beam/model/standard_coder_test_cases.yaml`.